ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -399,7 +399,11 @@ export const WorkspaceEdit = fp.flow(
           profile: { freeTierDollarQuota, freeTierUsage },
         },
       } = this.props;
-      const initialCreditsBalance = freeTierDollarQuota - freeTierUsage;
+
+      const freeTierUsageInNumber =
+        typeof freeTierUsage === 'undefined' ? 0 : freeTierUsage;
+
+      const initialCreditsBalance = freeTierDollarQuota - freeTierUsageInNumber;
```

Comment:
I think that this might be able to simplified to:

`const initialCreditsBalance = freeTierDollarQuota - (freeTierUsageInNumber ?? 0);`

or you could adjust what you have and change freeTierUsageNumber to:
`const freeTierUsageInNumber =
        freeTierUsage ?? 0;`

Reply:
thanks. What shall I do in `initial-credits-panel.tsx` page to have similar things? I don't think I can put that code snippet inside `FlexRow`

Reply:
In initial-credits-panel.tsx, you should be able to do something like this:
```
      <div style={{ marginTop: '0.6rem', fontWeight: 600 }}>
        {formatInitialCreditsUSD(props.freeTierUsage ?? 0)}
      </div>
      <div style={{ fontWeight: 600 }}>
        {formatInitialCreditsUSD(
          props.freeTierDollarQuota - (props.freeTierUsage ?? 0)
        )}
      </div>
```